packages/-ember-data/tests/integration/adapter/rest-adapter-test.js

```diff
@@ -313,8 +313,19 @@ module('integration/adapter/rest_adapter - REST Adapter', function (hooks) {
   });
 
   test('updateRecord - hasMany relationships faithfully reflect simultaneous adds and removes', async function (assert) {
-    Post.reopen({ comments: DS.hasMany('comment', { async: false }) });
-    Comment.reopen({ post: DS.belongsTo('post', { async: false }) });
+    Post = DS.Model.extend({
+      name: DS.attr('string'),
+      comments: DS.hasMany('comment', { async: false }),
+    });
+
+    Comment = DS.Model.extend({
+      name: DS.attr('string'),
+      post: DS.belongsTo('post', { async: false }),
+    });
```

Comment:
This is nice!  Btw, while we are here, should we move to `class Comment extends Model {`?

Reply:
This was left over from commits in a separate PR included by accident, but yes, starting out new code in a nice state is preferable 👍🏻